### DISCUSSION TITLE
[ARCropolis Version 3.6.2] Additional Aegis Slots Crash

### DISCUSSION THREAD

**Discussion by Blaze240** [Raytwo/ARCropolis#413] [Issues]
System Firmware Version
14.1.2

Atmosphere Version
1.3.2

Smash Version
13.0.1

ARCropolis Version
3.6.2 

Can the crash be consistently reproduced?
Yes

At what moment does the crash happen?
After pressing Start to progress beyond the CSS screen with an additional Aegis slot selected, screenshot below (or what can pass for one). No other mods enabled. Only plugin enabled is ARCropolis.

Details:
This report is purely for tracking purpose and requested by Raytwo. Also has an edited prc to extend the col_num index for all entries related to Aegis.
![01662146090_01006a800016e000](https://user-images.githubusercontent.com/107436977/188222294-8c8b14a6-99da-4092-9c80-89d696d4ea1b.jpg)
[01662146090_01006a800016e000.log](https://github.com/Raytwo/ARCropolis/files/9480566/01662146090_01006a800016e000.log)

**Comment by Raytwo** [comment]
Issue opened [here](https://github.com/Raytwo/ARCropolis/issues/415), thanks for reporting
